packages/utils/src/error.ts

```diff
@@ -163,8 +163,11 @@ export function replaceAsymmetricMatcher(actual: any, expected: any, actualRepla
         actualReplaced,
         expectedReplaced,
       )
-      actual[key] = replaced.replacedActual
-      expected[key] = replaced.replacedExpected
+      if (actualValue !== replaced.replacedActual)
```

Comment:
But what if it's frozen but inner value is different?

Reply:
I found out the root cause of the problem and won't have that concern now